include/boost/geometry/extensions/gis/projections/impl/pj_init.hpp

```diff
@@ -204,8 +205,14 @@ inline parameters pj_init(R const& arguments, bool use_defaults = true)
 
     if (! s.empty())
     {
-        // TODO: IMPLEMENT SPLIT
-        pin.to_meter = atof(s.c_str());
+        std::size_t const divisor = s.find('/');
+        if (divisor == std::string::npos)
+            pin.to_meter = std::atof(s.c_str());
+        else
+        {
+            std::string const numerator(s.substr(0, divisor)), denominator(s.substr(divisor + 1));
+            pin.to_meter = std::atof(numerator.c_str()) / std::atof(denominator.c_str());
```

Comment:
We're talking about the string that comes from the second column, `to_meter`, of `pj_units[]` in `pj_units.hpp`, right? I wouldn't think of that as a string the user enters though of course it is possible. If someone accidentally adds an entry with `100/0` then is run-time really the right place to catch it? Surely it can be caught with some compile-time check on the string literals?